torch/_dynamo/guards.py

```diff
@@ -187,16 +201,10 @@ def EQUALS_MATCH(self, guard: Guard):
                 torch.Size,
                 torch.device,
                 torch.dtype,
-                np.int8,
-                np.int16,
-                np.int32,
-                np.int64,
-                np.uint8,
-                np.uint16,
-                np.uint32,
-                np.uint64,
-            ),
+            )
+            + np_types,
```

Comment:
Had to leave this check in here as in some cases numpy integer types will come through. Specifically `hf_Reformer` was not able to compile without it.